string          src_ip
AvetoFrame      frame_info
AvetoTimebase   time0
AvetoTimebase   time1
AvetoTimebase   time2
uint32          payload_offset
